x-pack/test/security_solution_endpoint/apps/integrations_feature_flag/endpoint_exceptions.ts

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
Copied this file from ../integrations. It is used in FF FTR config and once this feature is GA this file should be copied over to ../integrations.